src/components/equations/ItemEffects.test.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
+import { stubBlankGame } from "../../fakes.test";
+import { IPokemon } from "../Battles";
+import { ItemEffects } from "./ItemEffects";
+
+describe("ItemEffects", () => {
+    describe("capturePokemon", () => {
+        it("adds a new Pokemon to your party when your party has fewer than six Pokemon", (): void => {
+            // Arrange
+            const { fsp } = stubBlankGame();
+            const pokemon: IPokemon = fsp.equations.newPokemon({
+                level: 5,
+                title: "SQUIRTLE".split(""),
+            });
+            const itemEffects = new ItemEffects(fsp);
+
+            // Act
+            itemEffects.capturePokemon(pokemon);
+            itemEffects.capturePokemon(pokemon);
+
+            // Assert
+            expect(fsp.itemsHolder.getItem(fsp.storage.names.pokemonInParty)[0].title).to.be.equal(pokemon.title);
+            expect(fsp.itemsHolder.getItem(fsp.storage.names.pokemonInParty)[1].title).to.be.equal(pokemon.title);
+        });
+
+        it("does not add a new Pokemon to your party when your party has more than six Pokemon", (): void => {
```

Comment:
Tests look good. However, you're missing tests ensure the correct return `true`/`false` return value.

Since that's two or three more tests with very similar arranges and acts, I'd recommend extracting the common arrange logic to a helper function.